### DISCUSSION TITLE
Entity Relationship Diagram ability

### DISCUSSION THREAD

**Discussion by emzwurld0911** [DaxStudio/DaxStudio#1265] [General]
I'm very new to DAX Studio. I have an inherited databased which in Power BI Desktop shows an incredible amount of relationships. Is there a way that the relationships can be shown in DAX studio, since you can't export relationships from Power BI?  I need to understand where the relationships are coming from and in the PBI Desktop view it's visually difficult to interpret.

And if there is no way in DAX Studio, does anyone have any free ERD software tool recommendations?

**Comment by marcosqlbi** [comment]
You can use the View Metrics feature: https://daxstudio.org/docs/features/model-metrics/
The Relationships pane shows a list of the relationships in the model. The documentation of the information provided is here: https://docs.sqlbi.com/vertipaq-analyzer/excel-metrics/relationship
